build.sh

```diff
@@ -221,20 +225,7 @@ esac
 __BuildOS=$__HostOS
 __BuildType=Debug
 __CMakeArgs=DEBUG
-
-case $__HostOS in
-    FreeBSD)
-        __monoroot=/usr/local
-        ;;
-    OSX)
-        __monoroot=/Library/Frameworks/Mono.framework/Versions/Current
-        ;;
-    *)
-        __monoroot=/usr
-        ;;
-esac
-
-__referenceassemblyroot=$__monoroot/lib/mono/xbuild-frameworks
+__referenceassemblyroot="$(csharp FindMonoXBuildFrameworks.cs)"
```

Comment:
Gotcha. I'll revert this part since it seems unnecessary / controversial / not how we want to do it long-term anyway. Thanks for the feedback guys :)